### PR TITLE
Etv 3980 drop kogaio dependency

### PR DESCRIPTION
**Pull request by sergiuelisei**
Updates: 

- removed @ivoryio/kogaio dependency and moved the base theme + missing components
- updated react version to 17
- added the new colors and the new typography variants

### CODE REVIEW

File: src/ActivityIndicator/LoadBar.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css, keyframes } from 'styled-components'
+import { Box } from '../Box'
+
+export const LoadBar = ({ colors, ...props }) => (
+  <Box bg='pastel-blue' height='4px' position='relative' width={1} {...props}>
```
Comment (alsofronie) [inline src/ActivityIndicator/LoadBar.jsx:7]: Can't you use one of the new colors ? like etvasDark or something?

File: src/ActivityIndicator/LoadBar.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  width: 0;
+
+  ${loopColours};
+`
+
+LoadBar.propTypes = {
+  colors: PropTypes.arrayOf(PropTypes.string)
+}
+LoadBar.defaultProps = {
+  colors: ['info', 'alert', 'error']
```
Comment (alsofronie) [inline src/ActivityIndicator/LoadBar.jsx:56]: Same here, maybe 'statusInfo', etc...

File: src/ActivityIndicator/RunningBar.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    width: 200px;
+  }
+`
+
+RunningBar.propTypes = {
+  colors: PropTypes.object.isRequired
+}
+RunningBar.defaultProps = {
+  colors: {
+    background: 'pale-white',
```
Comment (alsofronie) [inline src/ActivityIndicator/RunningBar.jsx:60]:
```suggestion
    background: 'baseWhite',
```

File: src/ActivityIndicator/Spinner.jsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+Spinner.propTypes = {
+  colors: PropTypes.object,
+  size: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+}
+
+Spinner.defaultProps = {
+  colors: {
+    background: 'white',
+    primary: 'brand'
```
Comment (alsofronie) [inline src/ActivityIndicator/Spinner.jsx:126]:
```suggestion
    background: 'baseWhite',
    primary: 'brand'
```

File: src/Touchable/Touchable.jsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+        :hover,
+        :active {
+          opacity: ${activeOpacity};
+        }
+      `
+    case 'highlight':
+      return css`
+        :hover {
+          background-color: ${themed(`colors.${underlayColor}`, underlayColor)};
+          color: ${themed('colors.white')};
```
Comment (alsofronie) [inline src/Touchable/Touchable.jsx:97]:
```suggestion
          color: ${themed('colors.baseWhite')};
```

File: src/Typography/variants.js [hunk 6]
```diff
@@ -1,4 +1,100 @@
-//   fontSizes: ['11px','12px', '14px', '16px', '20px', '24px', '32px','48px']
+//    fontSizes: ['11px', '12px', '14px', '16px', '20px', '24px', '32px', '48px'],
```
Comment (alsofronie) [inline src/Typography/variants.js:1]: Leftover?

File: stories/ActivityIndicator.stories.jsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+export default {
+  title: 'Demo/ActivityIndicator',
+  component: ActivityIndicator
+}
+
+export const Spinner = () => (
+  <Flex alignItems='center' flexWrap='wrap'>
+    <Flex justifyContent='space-evenly' width={1}>
+      <ActivityIndicator
+        colors={{ background: 'white', primary: 'statusWarning' }}
```
Comment (alsofronie) [inline stories/ActivityIndicator.stories.jsx:13]: Please use new colors in storybook

**Review by alsofronie — APPROVED** [review]

**Review by hututudor — APPROVED** [review]

**Pull request merged**